src/twisted/newsfragments/8368.removal

```diff
@@ -0,0 +1 @@
+twisted.cred.credentials.UsernameHashedPassword is now deprecated.
```

Comment:
Can we add a little more color on what was wrong with it / why it was deprecated? I don't want folks copy/pasting this into their own codebases, since it's not just moved or renamed, it's kind of backwards